core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -291,9 +291,6 @@ private[spark] class ExecutorAllocationManager(
    */
   private def schedule(): Unit = synchronized {
     val now = clock.getTimeMillis
-
-    updateAndSyncNumExecutorsTarget(now)
```

Comment:
IIUC the code is arranged intentionally to avoid ramp down initial executors until first job is submitted/idle executor is timeout, also cc @jerryshao